olm/open-liberty-crd.yaml

```diff
@@ -16,4 +16,463 @@ spec:
+                  type: string                
+                # Not sure if we should applied min/max restrictions...defaults are 2s,5s for readiness 20s,5s for liveness in the _deployment.tpl
+                readinessProbe:
+                  properties:
+                    failureThreshold:
+                      type: integer
+                      minimum: 1    
+                    initialDelaySeconds: 
+                      type: integer                                        
+                    periodSeconds: 
+                      type: integer
+                      minimum: 1
+                    successThreshold:
+                      type: integer
+                      minimum: 1                      
+                    timeoutSeconds:
+                      type: integer
+                      minimum: 1
+                  type: object   
+                repository:  
+                  type: string                    
+                security:
+                  type: object
+                serverOverridesConfigMapName:
+                  type: string        
+                tag:
+                  type: string
+              required:  
+              - pullPolicy
+              - repository
+              - tag
+              type: object
+            ingress:
+              properties:  
+                annotations: 
+                  type: object                  
+                enabled:
+                  type: boolean
+                host:
+                  type: string                  
+                labels: 
+                  type: object    
+                path:  
+                  type: string
+                rewriteTarget:
+                  type: string                  
+                secretName:
+                  type: string
+              required:           
+              - enabled
+              - path
+              - rewriteTarget
+              type: object  
+            jmsService:
+              properties:  
+                enabled:
+                  type: boolean
+                port:
+                  type: integer
+                targetPort:
+                  # values-metadata.yaml -- implies this is a string, but values.yaml and full_cr.yaml imply integer.
+                  type: integer                  
+                type:
+                  enum:
+                  - ClusterIP
+                  - NodePort
+                  type: string
+              required:           
+              - enabled
+              - port
+              - type
+              type: object              
+            logs:
+              properties:  
+                consoleFormat:
+                  enum:
+                  - json
+                  - basic
+                  type: string 
+                consoleLogLevel:
+                  enum:
+                  - audit
+                  - error
+                  - info
+                  - "off"
```

Comment:
It's odd that `"off"` is in quotes - was that intentional?

Reply:
I'll add a comment to explain "off".   It looks like it's treated as keyword for a boolean value.  So, without the quotes it fails validity.